components/user/UserEvents.js

```diff
@@ -1,46 +1,79 @@
 import { useState } from "react";
-
 import EventCard from "@components/event/EventCard";
 import Alert from "@components/Alert";
-import EventKey from "@components/event/EventKey";
```

Comment:
Is `EventKey` no longer used anywhere? If not used any where we should delete the file so we don't have any dead code

Reply:
Upon my exploration of the component folder, I found EventKey component is used nowhere. So, deleted the file as you said so that we don't have any dead code :smile: